docs/sources/whatsnew/whats-new-in-v7-3.md

```diff
@@ -15,14 +15,13 @@ weight = -17
 
 This topic includes the release notes for Grafana v7.3. For all details, read the full [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
 
-## Highlights
-
-Grafana 7.3 comes with a number of features and enhancements:
+The main highlights are:
 
 - [**Google Cloud Monitoring:** Out of the box dashboards]({{< relref "#cloud-monitoring-out-of-the-box-dashboards" >}})
 - [**Shorten URL for dashboards and Explore**]({{< relref "#shorten-url-for-dashboards-and-explore" >}})
+- [**SigV4 Authentication for Amazon Elasticsearch Service**]({{< relref "#sigv4-authentication-for-aws-users" >}})
 
-#### Cloud monitoring out-of-the-box dashboards
+## Cloud monitoring out-of-the-box dashboards
 
 The updated Cloud monitoring data source is shipped with pre-configured dashboards for five of the most popular GCP services:
```

Comment:
Please specify Google Cloud or Grafana Cloud to avoid confusion. @richardqlam and @matthewhelmke , something to watch for.

Reply:
Spell out what GCP stands for on the first usage.